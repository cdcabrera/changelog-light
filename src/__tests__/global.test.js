const global = require('../global');

describe('Global', () => {
  it('should return specific properties', () => {
    expect(global).toMatchSnapshot('specific properties');
  });
});
