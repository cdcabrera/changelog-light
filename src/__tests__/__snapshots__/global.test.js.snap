// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global should return specific properties: specific properties 1`] = `
{
  "color": {
    "BLACK": "[30m",
    "BLUE": "[34m",
    "CYAN": "[36m",
    "GREEN": "[32m",
    "GREY": "[90m",
    "MAGENTA": "[35m",
    "NOCOLOR": "[0m",
    "RED": "[31m",
    "WHITE": "[37m",
    "YELLOW": "[33m",
  },
  "contextPath": "../src/__fixtures__",
}
`;
